src/mmw/mmw/settings/gwlfe_settings.py

```diff
@@ -447,13 +447,26 @@
     'StreamFlowVolAdj': 1,  # Streamflow Volume Adjustment Factor
 }
 
+DAIRY_COWS = 'dairy_cows'
+BEEF_COWS = 'beef_cows'
+HOGS = 'hogs'
+SHEEP = 'sheep'
+HORSES = 'horses'
+BROILERS = 'broilers'
+LAYERS = 'layers'
+TURKEYS = 'turkeys'
+OTHER = 'other'
+# Animal keys ordered to mimic the order of the NumAnimals field
+NUM_ANIMALS_KEYS = [DAIRY_COWS, BEEF_COWS, BROILERS, LAYERS, HOGS, SHEEP, HORSES, TURKEYS, OTHER]
```

Comment:
Since these keys are used in more fields than just `NumAnimals`, consider naming this `ANIMAL_KEYS`, or even just `ANIMALS`.